less/common/Modal.less

```diff
@@ -105,6 +105,14 @@
       text-align: left;
     }
   }
+
+  .Checkbox--switch .Checkbox-display {
```

Comment:
I think if we use `.off.Checkbox--switch .Checkbox-display` here instead we could probably avoid having to declaring the block under it :smiley:

Reply:
Yep, just gave it a go - it works, thanks for the suggestion!